pkg/rancher-desktop/components/SortableTable/index.vue

```diff
@@ -1284,7 +1284,7 @@ $spacing: 10px;
 }
 
 .fixed-header-actions {
-  padding: 0 0 20px 0;
+  padding: 1px 0 20px 0;
```

Comment:
We might not want to add the padding directly within SortableTable, as it is copied over from Rancher Dashboard and we run the risk of reverting this change if we update SortableTable again in the future.

Reference: #2200